openshift/README.md

```diff
@@ -90,19 +96,19 @@ ports:
 #### Custom Datadog SCC for all features
 
 If SELinux is in permissive mode or disabled, enable the `hostaccess` SCC to benefit from all features.
-If SELinux is in enforcing mode, it is recommended to grant [the `spc_t` type][8] to the datadog-agent pod. In order to deploy the agent you can use the following [datadog-agent SCC][9] that can be applied after [creating the datadog-agent service account][6]. It grants the following permissions:
+If SELinux is in enforcing mode, it is recommended to grant [the `spc_t` type][11] to the datadog-agent pod. In order to deploy the agent you can use the following [datadog-agent SCC][12] that can be applied after [creating the datadog-agent service account][9]. It grants the following permissions:
 
 - `allowHostPorts: true`: Binds Dogstatsd / APM / Logs intakes to the node's IP.
 - `allowHostPID: true`: Enables Origin Detection for Dogstatsd metrics submitted by Unix Socket.
 - `volumes: hostPath`: Accesses the Docker socket and the host's `proc` and `cgroup` folders, for metric collection.
-- `SELinux type: spc_t`: Accesses the Docker socket and all processes' `proc` and `cgroup` folders, for metric collection. See [Introducing a Super Privileged Container Concept][8] for more details.
+- `SELinux type: spc_t`: Accesses the Docker socket and all processes' `proc` and `cgroup` folders, for metric collection. See [Introducing a Super Privileged Container Concept][11] for more details.
 
 <div class="alert alert-info">
 Do not forget to add a <a href="https://docs.datadoghq.com/agent/kubernetes/daemonset_setup/?tab=k8sfile#configure-rbac-permissions">datadog-agent service account</a> to the newly created <a href="https://github.com/DataDog/datadog-agent/blob/master/Dockerfiles/manifests/openshift/scc.yaml">datadog-agent SCC</a> by adding <code>system:serviceaccount:<datadog-agent namespace>:<datadog-agent service account name></code> to the <code>users</code> section.
 </div>
 
 <div class="alert alert-warning">
-<b>OpenShift 4.0+</b>: If you used the OpenShift installer on a supported cloud provider, you must deploy the Agent with <code>allowHostNetwork: true</code> in the <code>datadog.yaml</code> configuration file to get host tags and aliases. Access to metadata servers from the Pod network is otherwise restricted.
+<b>OpenShift 4.0+</b>: If you used the OpenShift installer on a supported cloud provider, you must deploy the SCC with <code>allowHostNetwork: true</code> in the <code>scc.yaml</code> manifest as well as <code>hostNetwork: true</code> in the Agent configuration to get host tags and aliases. Access to metadata servers from the Pod network is otherwise restricted.
```

Comment:
```suggestion
<b>OpenShift 4.0+</b>: If you used the OpenShift installer on a supported cloud provider, you must deploy the SCC with <code>allowHostNetwork: true</code> in the <code>scc.yaml</code> manifest, as well as <code>hostNetwork: true</code> in the Agent configuration to get host tags and aliases. Access to metadata servers from the Pod network is otherwise restricted.
```